dropbucket.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+if [ -z ${1} ] || [ -z ${2} ]; then
+  echo "Usage: $0 <zenko-name> <bucket-name> [-y]"
+  exit 1
+fi
+
+set -euo pipefail
+# args
+RELEASE=${1}
+BUCKET=${2}
+CONFIRM=${3:-false}
+# Sensitive collections to skip
+SKIP=(PENSIEVE __infostore __metastore __userbucket)
+PRIMARY="$(mongo --host ${MONGODB_REPLICASET} --quiet --eval="printjson(rs.isMaster().primary)" | sed 's/^"\(.*\)".*/\1/')"
+DB='metadata'
+
+confirm() {
+  if [ ${CONFIRM} == '-y' ]; then
+    return 0
+  else 
+    local yes=(y yes Yes YES)
+    echo "Warning: Confirm emptying '${BUCKET}' in Zenko '${RELEASE}' and all contained metadata? (y/n)"
```

Comment:
should the warning message also explicitly say that the action will lead to data orphans?

Reply:
I assume most people would be using this to reset OOB operations? but agree with Alex, good to be explicit

Reply:
Yeah you and @JianqinWang are right maybe some more details on the implications for the un initiated.